src/dmd/dinterpret.d

```diff
@@ -4997,7 +4997,7 @@ public:
         if (!fd.fbody)
         {
             e.error("`%s` cannot be interpreted at compile time, because it has no available source code", fd.toChars());
-            result = CTFEExp.cantexp;
+            result = new CTFEErrorExtraInfoExp();
```

Comment:
(2)... I think you can just `return CTFEExp.cantexp;` here...

Reply:
CTFEExp.cantexp can't be used because the ctfe code usually returns when cantexp is encountered and we need to transport the extra information outside the ctfe context.